src/components/form/RadioButton.vue

```diff
@@ -1,8 +1,9 @@
 <template>
-    <label class="ff-radio-btn" @click="select(value)">
+    <label class="ff-radio-btn" :disabled="disabled" @click="select(value)">
         <input type="radio" :value="value" />
```

Comment:
This should probably also have the disabled tag, it already _shouldn't_ be selectable as the click event is captured, but adding the tag too will improve accessibility and also prevent accidental keyboard navigation.